.github/workflows/e2e.yml

```diff
@@ -170,3 +170,9 @@ jobs:
         uses: ./.github/actions/workflow-notify
         with:
           slack-webhook: ${{ secrets.E2E_REGULAR_SLACK_WEBHOOK }}
+
+  e2e_dev1_x:
```

Comment:
Let's move it to the separate workflow with its own schedule. This one will be triggered on things like merge to `master` for instance that doesn't matter for `dev/1.x`.

Reply:
what exactly you would like to separate out?

Right now `e2e_dev1_x` is just a job inside `e2e` workflow that reuses `e2e` workflow defined in `ss/checkout-before-integration` branch (later `dev/1.x`).

I do it this way because there is no way to schedule time based triggers on non-default branches, so stuff like

```
  schedule:
    - cron: "0 4 * * *" # Run every day at arbitrary time (4:00 AM UTC)
```
will only work on `master` branch defined workflows.